packages/server/src/membership/membership.entity.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  BaseEntity,
+  Entity,
+  PrimaryKey,
+  Property,
+  Unique,
+  Collection,
+  OneToMany,
+  OneToOne,
+} from '@mikro-orm/core';
+import { User } from '../user/user.entity';
+import { Invoice } from '@server/invoice/invoice.entity';
+
+@Entity()
+//@Unique({ properties: ['user'] })
```

Comment:
A `OneToOne` relationship is unique by implication, so this wouldn't be necessary.